entgql/schema.go

```diff
@@ -25,6 +26,11 @@ import (
 	"github.com/vektah/gqlparser/v2/formatter"
 )
 
+var (
+	// ErrRelaySpecDisabled is the error returned when the relay specification is disabled
+	ErrRelaySpecDisabled = errors.New("entgql: must enable relay specification via the WithRelaySpecification option")
```

Comment:
Where do you see `WithRelaySpecification`? Also, better to keep it shorter as `WithRelaySpec`.

Reply:
Not expose the option yet, because it is still enabled by default.